MMA Removal Utility/SingleTenantSetupInstallation.md

```diff
@@ -60,8 +60,8 @@ Set-Prerequisites
 
 ### **Step 3 of 5. Setup remediation user-assigned managed identity**  
 The AzTS MMA Removal Utility solution works in 2 phases: 
-1. Discovers VMs and VM Extensions to identify the VMs from which MMA Agent will be removed for which setup requires ***Reader*** access on the scope being configured.
-2. Removes MMA Extensions from the eligible VMs for which setup requires ***Virtual Machine Contributor*** access on the scopes being configured. Scopes Configured can be a Tenant/ManagementGroup(s)/Subscription(s) or both ManagementGroup(s) and Subscription(s).
+1. Discovers VMs, VMSS and Azure Arc Servers and there Extensions to identify the services from which MMA Agent will be removed for which setup requires ***Reader*** access on the scope being configured.
```

Comment:
plz check if we need to add any new module or version as pre-requisite.

Reply:
No module addition required.